OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -2491,6 +2491,20 @@ namespace OloEngine
         }
     }
 
+    // Helper: obtain the shadow VAO RendererID from a Mesh (returns 0 if unavailable).
+    [[nodiscard]] static RendererID GetShadowVaoID(const Ref<Mesh>& mesh)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)